lib/minitest/reporters/base_reporter.rb

```diff
@@ -86,8 +86,12 @@ def print(*args)
       end
 
       def print_info(e, name=true)
-        print "#{e.exception.class.to_s}: " if name
-        e.message.each_line { |line| print_with_info_padding(line) }
+        if name
+          print "#{e.exception.class.to_s}: "
+          e.exception.message.each_line { |line| print_with_info_padding(line) }
+        else
+          e.message.each_line { |line| print_with_info_padding(line) }
```

Comment:
I'm confused :(  As far as I understand we print e.exception.message to avoid printing backtrace which is in e.message.  Am I right?  If so why name is false we print e.message?  Am I missing something?